lnbits/core/services.py

```diff
@@ -196,20 +197,24 @@ async def pay_invoice(
             invoice.amount_msat / 1000, wallet_id, extra=extra, conn=conn
         )
 
+        invoice_expiry = get_bolt11_expiry(payment_request)
+
         # put all parameters that don't change here
         class PaymentKwargs(TypedDict):
             wallet_id: str
             payment_request: str
             payment_hash: str
             amount: int
             memo: str
+            expiry: Optional[datetime.datetime]
             extra: Optional[Dict]
 
         payment_kwargs: PaymentKwargs = PaymentKwargs(
             wallet_id=wallet_id,
             payment_request=payment_request,
             payment_hash=invoice.payment_hash,
             amount=-invoice.amount_msat,
+            expiry=invoice_expiry,
```

Comment:
- nitpick: is a variable for `invoice_expiry` required?
```suggestion
            expiry=get_bolt11_expiry(payment_request),
```